flexget/tests/test_rtorrent.py

```diff
@@ -67,7 +67,10 @@ def test_load(self, mocked_proxy):
 
         fields = [p for p in called_args[2:]]
         assert len(fields) == 3
-        assert 'd.directory.set=\\/data\\/downloads' in fields
+        assert ('d.directory.set=\\/data\\/downloads' in fields
```

Comment:
You could use implementation independent test:
```python
assert ('d.directory.set=' + re.escape('/data/downloads')) in fields
```

Reply:
Good idea, done.